portfolio/src/main/webapp/index.html

```diff
@@ -49,6 +49,10 @@ <h1>This is Me!</h1>
             <p> Get some message from the Data Servlet ;P</p>
             <button onclick="getData()">Let's see the data!</button>
             <div id="data-container"></div>
+
+            <p> Array about to print (; </p>
+            <button onclick="getArray()">Let's see the array!</button>
```

Comment:
I just realized this function was added in the last PR, but we are using it in this PR. Somehow I missed that when reviewing before, but in the future it will be easier to review if you defined the function in the PR.

Reply:
Okay, noted for the future!